samples/robot/src/main/kotlin/example/robot.kt

```diff
@@ -24,14 +26,7 @@ fun main() = runMosaicBlocking {
 			Text("Position: $x, $y   World: $width, $height")
 			Text("")
 			// TODO https://github.com/JakeWharton/mosaic/issues/11
-			Text(buildString {
-				// TODO https://github.com/JakeWharton/mosaic/issues/7
-				repeat(y) { append('\n') }
-				repeat(x) { append(' ') }
-				append("^_^")
-
-				repeat(height - y) { append('\n') }
-			})
+			Text("^_^", modifier = Modifier.padding(x, y, width - x, height - y))
```

Comment:
Padding on the right is not really necessary, but without it world will not be beautiful if it's colored, like this
<img width="200" alt="image" src="https://github.com/JakeWharton/mosaic/assets/37583380/83e6bf87-4bdc-4392-a2bb-cf315705f28d">